teamcity: failed tests on master: check/TestStyle, check/TestStyle/TestErrCheck, check/TestStyle/TestReturnCheck, check/TestStyle/TestMetacheck, acceptance/TestDockerMigration_7429

The following tests appear to have failed:

[#209032](https://teamcity.cockroachdb.com/viewLog.html?buildId=209032):

```
--- FAIL: check/TestStyle (22.920s)
null
--- FAIL: check/TestStyle/TestErrCheck (7.050s)
style_test.go:570: err=exit status 2, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/migrations/main_test.go:28:2: SetReadFileFn not declared by package security
	error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/migrations_test
--- FAIL: check/TestStyle/TestReturnCheck (8.820s)
style_test.go:594: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/migrations/main_test.go:28:2: SetReadFileFn not declared by package security
--- FAIL: check/TestStyle/TestMetacheck (7.050s)
style_test.go:773: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/migrations/main_test.go:28:2: SetReadFileFn not declared by package security
	2017/04/06 05:15:51 couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/migrations_test
--- FAIL: acceptance/TestDockerMigration_7429 (7.140s)
reference_test.go:41: non-zero exit code: 1
------- Stdout: -------
test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerMigration_7429130250647  (use -show-logs to present inline)
test logs left over in: &{/home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerMigration_7429130250647 false}

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/migrations/main_test.go
+++ b/pkg/migrations/main_test.go
@@ -12,22 +12,15 @@
 // implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package migrations_test
+package migrations
 
 import (
-	"os"
-	"testing"
-
 	"github.com/cockroachdb/cockroach/pkg/security"
 	"github.com/cockroachdb/cockroach/pkg/security/securitytest"
-	"github.com/cockroachdb/cockroach/pkg/server"
-	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 )
 
-func TestMain(m *testing.M) {
-	security.SetReadFileFn(securitytest.Asset)
-	serverutils.InitTestServerFactory(server.TestServerFactory)
-	os.Exit(m.Run())
+func init() {
+	security.SetAssetLoader(securitytest.EmbeddedAssets)
 }
 
 //go:generate ../util/leaktest/add-leaktest.sh *_test.go

